src/mono/mono/mini/interp/transform.c

```diff
@@ -1383,9 +1383,7 @@ interp_generate_mae_throw (TransformData *td, MonoMethod *method, MonoMethod *ta
 		call_args [2] = -1;
 		td->last_ins->info.call_info->call_args = call_args;
 	} else {
```

Comment:
remove `{ }` as it contains single expression

Reply:
In general we use the convention that if any of the conditional blocks are encapsulated in parenthesis then all other blocks should be as well, even if they have a single instruction